src/actions/settings/currentNetworkActions.js

```diff
@@ -2,13 +2,45 @@ import { createActions } from 'spunky';
 import { getStorage, setStorage } from '../../lib/storage';
 
 export const ID = 'currentNetwork';
+export const NETWORKS_ID = 'networks';
```

Comment:
This constant and all related functions should go into a new file called `networksActions` since it's completely unrelated data that it's storing.